std/random.d

```diff
@@ -1270,6 +1270,19 @@ alias Xorshift    = Xorshift128;                            /// ditto
     }
 }
 
+version (OpenBSD)
+{
+    version = ARC4_RANDOM;
+}
+else version (NetBSD)
+{
+    version = ARC4_RANDOM;
+}
+
```

Comment:
I suggest also setting `version = ARC4_RANDOM` for `version (FreeBSD)` and `version (OSX)`.